templates/gitlab/mender-artifact-build.gitlab-ci.yml

```
@@ -0,0 +1,27 @@
+# GitLab CI template to build Mender artifacts.
+#
+# The template is designed to be used with the includes keyword or copied into an existing
+# configuration. It can be used to build a Mender artifact in the `script` section. 
+# `before_script` is used for creating the `MENDER_ARTIFACTS_FOLDER` folder. If it's overwritten,
+# then make sure your script contains this step.
+#
+# Mender artifacts in the `MENDER_ARTIFACTS_FOLDER` folder will be uploaded as GitLab CI artifacts
+# and shared across jobs in a pipeline.
+
+include:
+  - project: 'Northern.tech/Mender/mender-ci-workflows'
+    file: 'templates/gitlab/common.gitlab-ci.yml'
+    ref: 'master'
+
+.mender:build:artifact:
+  image: mendersoftware/mender-ci-tools:mender-master
+  before_script:
+    - if [ ! -d ${MENDER_ARTIFACTS_FOLDER} ]; then mkdir -p ${MENDER_ARTIFACTS_FOLDER}; fi
+  script:
+    # to be overwritten after the job is inherited
+    - echo "Please override this job in your pipeline"
```

```suggestion
    - echo "Please override the script keyword in your job"
```

What do you think? it is not a "job"

---

What do you mean? It can still be improved even I originally suggested it :shrug: